ParisBNF/abb/BNFabb105.xml

```diff
@@ -834,13 +834,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           <list>
                           <item xml:id="a1">
                             <locus from="1v" to="3r"></locus>
-                             <desc type="GuestText" corresp="LIT6011FeastsList" cert="medium"/>
+                            <desc type="GuestText"><title type="complete" ref="LIT6011FeastsList" cert="medium"></title>
```

Comment:
`<title type="complete" ref="LIT6011FeastsList" cert="medium">List of Feasts</title>`